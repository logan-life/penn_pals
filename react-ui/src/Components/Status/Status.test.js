import React from 'react';
import StatusView from './Status';
import {BrowserRouter} from "react-router-dom";

const renderer = require('react-test-renderer');

let container;

beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  document.body.removeChild(container);
  container = null;
});

// snapshot testing
test('StatusView matches snapshot', () => {
const component = renderer.create(<BrowserRouter><StatusView /></BrowserRouter>);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});
