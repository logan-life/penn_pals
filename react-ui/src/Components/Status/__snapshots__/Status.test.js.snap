// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusView matches snapshot 1`] = `
<div
  className="fade alert alert-danger show"
  id="alertContactViewNotLoggedIn"
  role="alert"
>
  Please
   
  <a
    className="alert-link"
    href="/Login"
    onClick={[Function]}
  >
    login
  </a>
   
  to see your status.
</div>
`;
